advanced-api/dynamic-remotes-runtime-environment-variables/e2e/checkDynamicRemotesRuntimesApps.cy.ts

```diff
@@ -0,0 +1,104 @@
+        loading: Constants.elementsText.dynamicSystemRemotesRuntime.loading,
+        buttonHeader: Constants.elementsText.dynamicSystemRemotesRuntime.buttonHeader,
+        buttonH2: Constants.elementsText.dynamicSystemRemotesRuntime.buttonH2,
+        buttonParagraph: Constants.elementsText.dynamicSystemRemotesRuntime.buttonParagraph,
+        host: 3001
+    }
+]
+
+appsData.forEach((
+    property: {
+        header: string,
+        subheader: string,
+        hostH3?: string,
+        paragraph?: string,
+        button?: string,
+        loading: string,
+        buttonHeader: string,
+        buttonH2?: string,
+        buttonParagraph: string,
+        host: number
+    }
+) => {
+    describe(`Check Host and Remote apps`, () => {
+        beforeEach(() => {
+            basePage.openLocalhost(property.host)
+        })
+        it (`Check host and Remote apps starts and running`, () => {
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h1,
+                text: property.header
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h2,
+                text: property.subheader
+            })
+            if (property.host === 3000) {
+                basePage.checkElementWithTextPresence({
+                    selector: baseSelectors.h3,
+                    text: property.hostH3
+                })
+                basePage.checkElementWithTextPresence({
+                    selector: baseSelectors.paragraph,
+                    text: property.paragraph
+                })
+                basePage.clickElementWithText({
+                    selector: baseSelectors.button,
+                    text: String(property.button)
+                })
+            }
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.divElement,
+                text: property.loading
+            })
+            cy.wait(2000)
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h2,
+                text: property.buttonHeader
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h2,
+                text: property.buttonH2
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.paragraph,
+                text: property.buttonParagraph
+            })
+            if (property.host ===3000) {
+                basePage.checkElementWithTextPresence({
+                    selector: baseSelectors.paragraph,
+                    text: getDateWithFormat('current', 'MMMM Do YYYY, h:mm:ss a')
+                })
+            }
+            else basePage.checkElementWithTextPresence({
+                selector: baseSelectors.paragraph,
+                text: getDateWithFormat('current', 'MMMM Do YYYY, h:mm:ss a')
+            })
```

Comment:
I think you don't need to similar methods one in if second in else
You can left one of them here